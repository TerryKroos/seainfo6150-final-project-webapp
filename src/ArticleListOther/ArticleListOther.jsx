import React from 'react';
import ArticleItemOther from "../ArticleItemOther/ArticleItemOther.jsx";
import styles from "./ArticleListOther.module.css";


const ArticleListOther = (props) => {
  
    return (
      <div className={styles.container}>
        <ul className={styles.articles}>
          {props.articles.map((article) => (
            <ArticleItemOther article={article} key={article.slug} />
          ))}
        </ul>
        </div>
    );
  };

  export default ArticleListOther;