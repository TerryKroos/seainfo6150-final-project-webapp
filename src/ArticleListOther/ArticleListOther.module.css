.container {
    display: flex;
    justify-content: center;
    background: #BBBBBB;
}
.articles {
    border-radius: 10px;
    display: flex;
    flex-direction: column;
    font-family: 'Inter', sans-serif;
    list-style-type: none;
    padding: 15px;
    margin: 35px;
  }
  
  @media screen and (min-width: 525px) {
    
  }
  
  @media screen and (min-width: 800px) {
    
  }