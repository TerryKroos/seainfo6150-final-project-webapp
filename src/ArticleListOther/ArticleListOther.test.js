
import React from "react";
import ArticleListOther from "./ArticleListOther";
import { render } from "@testing-library/react";

describe("ArticleListOther tests", () => {
    it("renders correctly", () => {
      const { container } = render(<ArticleListOther articles={[]}/>);
      expect(container).toMatchSnapshot();
    });
  
  });