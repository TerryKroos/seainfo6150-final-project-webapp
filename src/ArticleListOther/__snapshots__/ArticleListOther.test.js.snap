// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleListOther tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <ul
      class="articles"
    />
  </div>
</div>
`;
